src/main/resources/jenkins/security/plugins/ldap/LDAPConfiguration/config.jelly

```diff
@@ -47,5 +47,8 @@
                 </f:entry>
             </f:repeatableProperty>
         </f:entry>
+        <f:entry field="ignoreIfUnavailable" title="${%Ignore if Unavailable}">
+            <f:checkbox />
+        </f:entry>
     </f:advanced>
 </j:jelly>
```

Comment:
just a line break at the end could be nice

Reply:
Since this isn't a new file, my preference is to just leave it as-is and handle it in a formatting-only PR later if desired. Is that ok?

Reply:
My concern was more concerning the help text, so no problem